src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
```

Comment:
it's not a good practice to reset values like this (using `*` selector). 
Reset `margin` and `padding` for tag selectors instead